modules/auxiliary/scanner/mysql/mysql_file_enum.rb

```diff
@@ -42,11 +44,9 @@ def mysql_query_no_handle(sql)
   end
 
   def run_host(ip)
-    vprint_status("Login...")
+    vprint_status("Login...") unless session
```

Comment:
Does this mean this module requires a session?

Reply:
The idea here was to add logic to handle whether to print whether we're logging in or not. If we have a session, we are already logged in, so we don't want to output that.

e.g.
![image](https://github.com/rapid7/metasploit-framework/assets/69522014/70a252d0-5411-43b1-9dc1-3384bec2b4b7)